tests/integration/test_refreshable_mat_view_replicated/test.py

```diff
@@ -357,124 +358,6 @@ def compare_DDL_on_all_nodes():
     compare_DDL_on_all_nodes()
 
 
-@pytest.mark.parametrize(
-    "append",
-    [True, False],
-)
-@pytest.mark.parametrize(
-    "empty",
-    [True, False],
-)
-@pytest.mark.parametrize(
-    "to_clause",
-    [
-        (None, "tgt1", "tgt1"),
-        ("Engine ReplicatedMergeTree ORDER BY tuple()", None, "test_rmv"),
-    ],
-)
-def test_real_wait_refresh(
```

Comment:
(This test was removed from the other copy-pasta file a while ago, but forgotten here.)